evap/development/fixtures/test_data.json

```diff
@@ -153476,7 +153476,8 @@
   "fields": {
     "name": "Big party",
     "date": "2099-12-31",
-    "redeem_end_date": "2099-12-30"
+    "redeem_end_date": "2099-12-30",
+    "step": 1
```

Comment:
This should be something other than 1 to have more useful testdata

Reply:
Let's add a comment to `RewardPointRedemptionEvent.step` saying something like

> Note that we allow this value to change throughout the lifetime of the event.